app/modules/assets/schemas.py

```diff
@@ -67,6 +67,18 @@ class Meta(BaseAssetSchema.Meta):
         )
 
 
+class ExtendedAssetDetailedAnnotationsSchema(BaseAssetSchema):
```

Comment:
If instead you'd inherited from ExtendedAssetSchema, you could have lost the class Meta as it's identical